README.md

```diff
@@ -265,6 +265,19 @@ by the chosen storage driver's provider. A list of drivers can be found [here](h
 [Horizontal](https://docs.openshift.com/container-platform/latest/nodes/pods/nodes-pods-autoscaling.html) and
 [Vertical](https://docs.openshift.com/container-platform/latest/nodes/pods/nodes-pods-vertical-autoscaler.html) Pod
 autoscaling support are not available for Windows workloads.
+
+### Node reboot in AWS
+
+In AWS, Windows nodes are not able to become ready after [performing a graceful reboot](docs/TROUBLESHOOTING.md#rebooting-a-windows-node) 
+due to an inconsistency with the EC2 instance metadata routes and the HNS networks.
+
+The cluster administrator must SSH into the instance and add the route after rebooting with the following command:
+```cmd
+  route add 169.254.169.254 mask 255.255.255.255 <gateway_ip>
```

Comment:
Good work documenting this workaround. Unfortunately this seems like a big ask for admins.
I'm not opposed to merging it but I'd like to see if there's anything else you find that can be done on our side.

Reply:
>  seems like a big ask for admins.

In this context, Windows admins require/need to log into the VM (via SSH for example) to update system components that require a reboot, then trigger the reboot. IMO, this suggestion just add another step (add the AWS routes) to the existing workflow, where complexity remains the same.

Reply:
> Anything else you find that can be done on our side.

- https://github.com/openshift/windows-machine-config-operator/pull/2282

was the initial proposal.